lib/movable_ink/aws/sns.rb

```diff
@@ -77,8 +77,9 @@ def notify_pagerduty(region:, instance_id:)
       end
 
       def add_subject_info(subject:)
-        required_info = " (#{instance_id}, #{my_region})"
-        "#{subject.slice(0, 99-required_info.length)}#{required_info}"
+        instance_link = "<https://#{my_region}.console.aws.amazon.com/ec2/v2/home?region=#{my_region}#Instances:search=#{instance_id};sort=instanceId|#{instance_id}>"
```

Comment:
Can we have a multi-line subject?

Reply:
Yep, no issues there.

This is the subject from one of the specs
<img width="652" alt="Screen Shot 2019-11-06 at 8 00 37 AM" src="https://user-images.githubusercontent.com/786640/68316640-698e4700-007f-11ea-9b68-554b02a19377.png">